src/main/java/org/shanerx/tradeshop/util/Updater.java

```diff
@@ -1,19 +1,24 @@
 /*
- *     Copyright 2016-2017 SparklingComet @ http://shanerx.org
+ *     Copyright (c) 2016-2017 SparklingComet @ http://shanerx.org
```

Comment:
Theoretically, (c) = © = Copyright, so we are repeating the same word twice.

On a serious note, though, I thought you were part of this copyright - I remember clearly adding you to the header.

Make sure to add yourself before the 2.0 release ^_^

Reply:
well see if i remember and also i didnt put that c there sooooo ya...

Reply:
Oh? Well, that’s unexpected... Not sure why’d appear there, then.